blocks/image-compare/src/edit.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls, RichText } from '@wordpress/block-editor';
+import { SelectControl, TextControl, Placeholder } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import UploadPlaceholder from './upload-placeholder';
+
+const edit = ( { attributes, className, isSelected, setAttributes } ) => {
+	const { imageBefore, imageAfter, caption, orientation } = attributes;
+
+	// if both are defined, add juxtaspose class
+	// the juxtapose library uses class when page is scanned
+	// to find the images and apply the side-by-side magic
+	const cls = ( imageBefore && imageAfter ) ? 'juxtapose' : 'controls';
+
+	return (
+		<>
+			<InspectorControls key="controls">
+				<h4> Orientation </h4>
```

Comment:
Extra spaces :) 

Also, text needs to be translatable.

Reply:
Fixed.